### ISSUE TITLE
Ability to hide nodes with no edges

### ISSUE THREAD

**Issue by Phiney89** [almende/vis#1135]
Is this already possible? Could it be possible?

**Comment by AlexDM0** [comment]
Hi,

All nodes and edges have an option called hidden. Use it :)

It's up to you to determine which nodes have no edges. 

Regards,

Alex

**Comment by RobertTheGrey** [comment]
@AlexDM0 - this hidden option works just fine! The problem is it triggers the physics when the node hides - is there any way to prevent this to stop the network bouncing around when you just hide a node?

Thanks!

**Comment by AlexDM0** [comment]
Hi, 

Which version are you using?

Regards

**Comment by RobertTheGrey** [comment]
Latest version - v4.7.0. Each time I hide (or remove) the node and then show (or add it back) it triggers a physics bounce of all connected nodes and then eventually settles again

**Comment by AlexDM0** [comment]
Hi,

This is interesting, it shouldn't bounce in the latest version.  There was a bug on this a while back hence the question. Could you make a jsbin to show this?

Regards

**Comment by RobertTheGrey** [comment]
Ah, I went ahead and built a simple jsbin hoping to reproduce the error and hiding and showing does not cause a bounce in jsbin. So I'll have to go through my code on Monday again and check what could be causing it. Just as long as I know it's not *supposed* to bounce then I can try and fix it. Thanks for coming back!

**Comment by RobertTheGrey** [comment]
FYI - the JSBIN is here: https://output.jsbin.com/rivukimiko

**Comment by RobertTheGrey** [comment]
Hi @AlexDM0 - I'm hesitating to raise a new issue, so rather just putting some new info here from our conversation last week. I've managed to reproduce the Bounce issue with v4.7.0 and is seems to be related to Scale.

Please see the JSBIN here: https://output.jsbin.com/tuvonadoqo

Click - hide node and you'll see it works fine and focuses + scales to the node I want perfectly. But when I hit show node again the Bounce happens. Is there any way to avoid this or is it by design?

As a side note - the bounce does not happen if you take the call t network.focus() out or if you keep the scale option = 1. But of course I want to scale to 2 and focus and hide the node in question, so is there a way to achieve that without the physics bounce?

Hopefully you have enough info - if not, please let me know.

**Comment by AlexDM0** [comment]
Hi,

I see, it seems you're in luck, the changes to the physics system in the next release take care of this issue. 

Regards

**Comment by RobertTheGrey** [comment]
Great news! :+1: - any idea on when that next release is due out? Or can I possibly use it now from the latest branch?

**Comment by AlexDM0** [comment]
I would wait on the release, its not fully tested yet. The release will probably be tomorrow, at the latest end of the week.

Regards

**Comment by RobertTheGrey** [comment]
Thank you! I'll wait
